src/components/dropdown_styles.css

```diff
@@ -0,0 +1,148 @@
+/* Dropdown Button */
```

Comment:
Lets have our files names in camel case to be consistent with amplify files and the work already done by other devs.